cli-output/src/cli_output.rs

```diff
@@ -378,22 +390,45 @@ impl fmt::Display for CliValidators {
             ))
             .bold()
         )?;
-        for validator in &self.current_validators {
-            write_vote_account(
-                f,
-                validator,
-                self.total_active_stake,
-                self.use_lamports_unit,
-                false,
-            )?;
+
+        let mut sorted_validators = self.validators.clone();
+        match self.validators_sort_order {
+            CliValidatorsSortOrder::Delinquent => {
+                sorted_validators.sort_by_key(|a| a.delinquent);
+            }
```

Comment:
Hmm, would it be helpful to have a default secondary sort for primary sorts like this that have a bunch of matching results? Or random okay?
Just thinking about this move to have important stuff at the bottom: https://github.com/solana-labs/solana/pull/16640/files#r616116236

Reply:
Here's what the new default output looks like for devnet:
```
$ solana validaors
    Identity                                      Vote Account                            Commission  Last Vote  Root Block Epoch Credits Version  Active Stake
 1⚠️ 6xDdF6YUAKVfj6AL3TMJckQokXhrxtBwMiuHVMnSHxR   GamfjAaa1y2vNbR54vfQM3Q5NrrQ5xCDtdoVC93jWiJ7  100%   35371461    35371430             0 unknown  0.99771712 SOL (0.00%)
 2⚠️ 2MeGu4yCwRCthwzFiVtsLAVSa6xCsdH59cz6F6J6b33U  GrnK4vPzYLzFdnirZCBXjLSvKG4iF6pHdZ67C9rZsbSG  100%   36055114    36055083             0 unknown  0.99771712 SOL (0.00%)
 3⚠️ AJib74LMUmCfkRN84iEWApuhGzEpQcA4g7KC1FVgqWCt  72h1eypUDXm6ks6X8uWbpuzKgRNZiCRSwnefFHjGyxEa  100%   46034764    46034733             0 unknown  0.99771712 SOL (0.00%)
 4⚠️ 4GskhQsHVuMv68guHRwr5f5DgL9vHEMcV1KfNcvS7VhG  Mx7uRGuYtxknJKuXQz3mv5EkiydRXeDXWDjNdgYmM8F   100%   50095833    50095802             0   1.6.4  0.99771712 SOL (0.00%)
 5⚠️ 8KeYrkYrNSq1x5zDQqghEiXFmf4QjYUWUzVt6qwiBX5i  775UhDftsU4SMEUHK8nCK1P5o4eGnoGwX6FdhBbA6LtC  100%   44829320    44829289             0 unknown  0.99771712 SOL (0.00%)
 6⚠️ 7QqYu69Sh5WB58JjXEmnuKVfvybC4dxWTsDFmNqiY9d4  2Y6BbUbkcEKdnAr9ED5bLBH6nC4XnfBUnjrVcRghozcw  100%   45329091    45329060             0 unknown  0.99843424 SOL (0.00%)
 7⚠️ 4mMCNHeEZwqkM99HAzQMbiPFCXhsC3VAo56BkFoTVHoY  7GHnTRB8Rz14qZQhDXf8ox1Kfu7mPcPLpKaBJJirmYj2    8%   48268420    48268389             0 unknown  1.001521874 SOL (0.00%)
 8  BmSmUWB9gJkoePfnmnKGtcG9MXnW1VKDvvTRman8F1Te  BGMDEqc5TZ3xkSKWeGtiAHPNkbc4GeBqnQdNJry2xGth    0%   50803654    50803623        259242   1.6.5  1.270443191 SOL (0.00%)
 9⚠️ 5cntEEYR5DMoDucsjQKCjHyhR7CDbytAeW1Y8osHAd3R  CrinLuHjVGDDcQfrEoCmM4k31Ni9sMoTCEEvNSUSh7Jg  100%   41470176    41470145             0 unknown  1.29315136 SOL (0.00%)
10⚠️ G7131eBQAa4NhX8V3JXRZwctZRraLMdUPcAAH1M5xpDo  7Lm4wRjaH7CvrzaD4oRrXFu6FFdCreYupFQHouh8Uojy  100%   35126559    35126528             0 unknown  1.88630272 SOL (0.00%)
11⚠️ 82BPXtNdKBhiEEFdB9Vm6dJpj6jGuoUxEKHxZWiBcgyX  BbLFfmDiViMAXZZf1eLHRtQAuFS1aG8PJ2AmE5NLm1mY   10%   49514508    49514477             0 unknown  2.035346098 SOL (0.00%)
12⚠️ E4aMcPhHEoyLxMW8jJgMkcyK1S6qNhjjZGsyhCY4UiAd  GemaKPJ7cWmVsqbb5T5z5MyDYLreRaUqv7MswYbmqgSV  100%   40239267    40239236             0 unknown  29.026114798 SOL (0.00%)
13⚠️ H9kSDiJJFVoLjpmue8nKhvLteZfzjuzS8AjUtBovxBZE  GNkBiAHKTk5RYxi1bSEKc31xibMqSBTKtLkRbXhi1h8K  100%   32427525    32427494             0 unknown  82.99543424 SOL (0.00%)
14⚠️ DctYdX8c3qBZ7RUtYE4Ffunjv5SYFxVde4H4BDejPzMG  HK456zW1akyEDBdmvcBddyvxfefGCvTBszKBakAZPR3S  100%   12223953    12223922             0 unknown  94.4846528 SOL (0.00%)
15⚠️ EUcJwf7jXskRE6NZBtFPVH2EedNvNYko8LL2WT62XctB  Bwkz1ddKoGE8hgiSV6HZLXi9RBLqfBi3HZb2QujzVGgz  100%   39257131    39257100             0 unknown  100.99543424 SOL (0.00%)
16⚠️ FSgwdAhy9JAePSKtrCrDFmMFpKwufd1WVBimHRc2dDEk  85ShayF5HDXzZxxey8Qd89VSCfBxCgtmuL4LJx6GTkr2    9%   44780479    44780448             0 unknown  149.24315136 SOL (0.00%)
17⚠️ 815YLNgAsnMqWJo1dmy13oKwWxujq31Dh6KhCUirnhBh  DTmhJTdXM8QBQ5p4mL2v6XzVqin5t9RdtFQMkXctu2La   10%   38045501    38045470             0 unknown  1013.495481318 SOL (0.00%)
18  dv1LfzJvDF7S1fBKpFgKoKXK5yoSosmkAdfbxBo1GqJ   5MMCR4NbTZqjthjLGywmeT66iwE9J9f7kjtxzJjwfUx2    0%   50803654    50803623        259654   1.6.6  77076339.979697436 SOL (100.00%)

Active Stake: 77077823.693751261 SOL
Current Stake: 77076341.250140622 SOL (100.00%)
Delinquent Stake: 1482.443610648 SOL (0.00%)

Stake By Version:
1.6.4    -   0 current validators ( 0.00%),   1 delinquent validators ( 0.00%)
1.6.5    -   1 current validators ( 0.00%)
1.6.6    -   1 current validators (100.00%)
unknown  -   0 current validators ( 0.00%),  15 delinquent validators ( 0.00%)
```

Is this confusing now that the only staked node is number 18?

Reply:
> Or random okay?

I'm game for random until somebody doesn't like it :)

Reply:
> Is this confusing now that the only staked node is number 18?

I'm still getting used to the enumeration generally, but imo it's okay.

Reply:
Disable the enumeration by default?  I reached for this initially but failed to come up with a nice short flag to enable it.  `--numbers` didn't feel right, and `--line-numbers` worse.  hmm, maybe `--count`?

Reply:
I like `--count`

Reply:
If the numbering is going to be optional, can we get an affirmative character in the `delinquent` column? This'll make `cut`ing in scripts easier.  I used to curse it for `sort`ing, but you've resolved that otherwise :slightly_smiling_face:

Reply:
> If the numbering is going to be optional, can we get an affirmative character in the `delinquent` column? 

What do you have in mind?

I'll add `--count` though

Reply:
Some precedents, `--number` (`cat`) and `--line-number` (`grep`) :shrug: 

`--count` seems fine so long as we don't see adding any pattern matching as likely (I don't)

Reply:
I took a look at `man cat` and liked what I saw wrt `--number`

Reply:
> What do you have in mind?

:heavy_check_mark: / `.` ?

Reply:
🤔 I kinda like the clean look of ` ` for current validators though

Reply:
Does f179221cddc82d1309bac90116149540ab3877b6 look good?

Reply:
It looks like what I tested with. :shipit: